src/pages/home/components/Tag.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react';
+import * as styles from '@/pages/home/components/Tag.css';
+import Text from '@shared/components/text/Text';
+import { TAG } from './constant/tags';
+
+const Tag = () => {
+  const [selectedTag, setSelectedTag] = useState<string>('recommend');
+
+  return (
+    <div className={styles.tagWrapper}>
+      {TAG.map(tag => (
+        <div
+          className={styles.tag({ selected: selectedTag === tag.id })}
+          key={tag.id}
+          onClick={() => setSelectedTag(tag.id)}
+        >
+          {tag.icon}
+          <Text
+            tag={selectedTag == tag.id ? 'body_bold_14' : 'body_regular_14'}
+            color={selectedTag == tag.id ? 'white' : 'black'}
+          >
+            {tag.text}
+          </Text>
+        </div>
+      ))}
```

Comment:
```suggestion
      {TAG.map(({id, text, icon}) => (
        <div
          className={styles.tag({ selected: selectedTag === id })}
          key={id}
          onClick={() => setSelectedTag(id)}
        >
          {icon}
          <Text
            tag={selectedTag == id ? 'body_bold_14' : 'body_regular_14'}
            color={selectedTag == id ? 'white' : 'black'}
          >
            {text}
          </Text>
        </div>
      ))}
```
이렇게 구조분해할당 이용해도 괜찮을것 같아요!

Reply:
찾아봤는데 flexWrap: 'nowrap'은 굳이 필요하지 않네요...
수정하겠습니다!